core/market/src/config.rs

```diff
@@ -23,7 +23,9 @@ pub struct DiscoveryConfig {
     pub mean_cyclic_unsubscribes_interval: Duration,
 }
 
+#[derive(StructOpt)]
 pub struct SubscriptionConfig {
+    #[structopt(env, parse(try_from_str = parse_chrono_duration), default_value = "1h")]
```

Comment:
I'd call the variable `DEFAULT_SUBSCRIPTION_TTL` or `SUBSCRIPTION_DEFAULT_TTL` (whichever fits `yagna` naming convention for config env vars better).